app/src/main/java/ca/bc/gov/bchealth/ui/home/immunizationschedules/ImmunizationSchedulesScreen.kt

```diff
@@ -0,0 +1,101 @@
+package ca.bc.gov.bchealth.ui.home.immunizationschedules
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.MyHealthTypography
+import ca.bc.gov.bchealth.compose.bold
+import ca.bc.gov.bchealth.compose.theme.bannerInfoBg
+import ca.bc.gov.bchealth.compose.theme.white
+import ca.bc.gov.bchealth.ui.custom.DecorativeImage
+import ca.bc.gov.bchealth.ui.home.immunizationschedules.ImmunizationSchedulesViewModel.ImmunizationSchedulesItem
+
+@Composable
+fun ImmunizationSchedulesScreen(
+    uiList: List<ImmunizationSchedulesItem>,
+    onClickItem: (String) -> Unit
+) {
+    LazyColumn(
```

Comment:
abstract this out in to private method

Reply:
in this case the LazyColumn is already the only thing being called in this method, so it doesn't make sense to extract to a method

Reply:
Hat will help you to put the viewModel call to get the list inside the screen block with launch effect, which you have to do eventually.

Reply:
done